ush/gfs_sndp.sh

```diff
@@ -19,18 +19,21 @@ cd $DATA/$m
     file_list=gfs_collective${m}.list
 
     if [ $m -le 2 ]
-    then 
+    then
       WMOHEAD=JUSA4$m
     elif [ $m -le 6 ]
-    then 
+    then
       WMOHEAD=JUSB4$m
-    else 
+    else
       WMOHEAD=JUSX4$m
     fi
 
     for stn in $(cat $file_list)
     do
-       cpreq "${COMOUT_ATMOS_BUFR}/bufr.${stn}.${PDY}${cyc}" "${DATA}/${m}/bufrin"
+       bufr_target="bufr.${stn}.${PDY}${cyc}"
+       cpreq "${COMIN_ATMOS_BUFR}/${bufr_target}" "${DATA}/${m}/bufrin"
+       # Also write the filename to a list for files to tar
+       echo "${bufr_target}" >> "${DATA}/${m}/bufr_snd_files.list"
```

Comment:
Do we still need this line to generate the filename list?

Reply:
@BoCui-NOAA Yes, you can find test case tar file here: `/lfs/h2/emc/eib/noscrub/david.huber/rt_gempak/COMROOT/C96_atm3DVar_extended_gempak/gfs.20211222/06/products/atmos/bufr/gfs.t06z.bufrsnd.tar.gz`